src/style.css

```diff
@@ -41,17 +44,55 @@ body {
   gap: 15px;
 }
 
-.scores-container {
+.score-wrapper {
   margin-top: 15px;
-  border: 1px solid black;
+  border: 2px #000 solid;
+  gap: 15px;
+}
+
```

Comment:
- [ ] Please kindly make adjustment to the spacing of your app; you may increase the width of this `container `so as to have a design close to the template. This will allow the app user to input their details conveniently with no constraint. 👍 
- [ ] Kindly increase the font size of the app title as well `Leaderboard`. ℹ️ 


## Your design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/211160823-25682771-72fb-45d8-94f4-38dd51975bcf.png)

## Wireframe
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/211160847-4366efda-b855-466c-9b46-71f443a37be9.png)